doc/api/fs.md

```diff
@@ -436,7 +436,7 @@ changes:
 -->
 
 * `path` {string|Buffer|URL}
-* `mode` {integer}
+* `mode` {integer} **Default:** `fs.constants.F_OK`
```

Comment:
@jasnell seems like an issue for https://github.com/nodejs/node/pull/13769, which I had thought you were okay with.